doc/cfgfile.rst

```diff
@@ -80,6 +81,13 @@ Application section
    If you use the Python API, this parameter can also be passed as a file-like
    object, such as :class:`io.StringIO`.
 
+.. describe:: license (optional)
```

Comment:
For the sake of explicitness, I like your earlier suggestion to call this `license_file` - that makes it clear that it's a path, not just a name like `GPL`.